### PR TITLE
Change contact info text

### PR DESCRIPTION
**Pull request by murrayholmes96**
Issue: Scottish-Natural-Heritage/Licensing#657

### CODE REVIEW

File: src/views/start.njk [hunk 1]
```diff
@@ -49,26 +49,44 @@
         }) }}
       </form>
       <p class="govuk-body">
-      If you need assistance with this service, contact the NatureScot Licensing Team.
+        If you need assistance with this application, contact the NatureScot
```
Comment (JohnJAMoore) [inline src/views/start.njk:52]: did this need to change?

**Review by JohnJAMoore — COMMENTED** [review]
Just a quick question 

this used to read "If you need assistance with this service, contact the NatureScot Licensing Team."

are we now referring this as a application rather than a service?

File: src/views/start.njk (hunk 1 above)
Comment (murrayholmes96) [reply src/views/start.njk:52]: nah this is my bad, ill fix

Comment (murrayholmes96) [reply src/views/start.njk]: Reverted the text that i changed by mistake

Comment (JohnJAMoore) [reply src/views/start.njk]: katherine would like this text to be - 'If you need assistance with this service, contact the NatureScot Licensing Team'

**Comment by JohnJAMoore** [comment]
Closing PR as i am re opening a different one see https://github.com/Scottish-Natural-Heritage/trap-registration-return/pull/61

**Pull request closed without merging**